tools/brl_checks.c

```diff
@@ -134,6 +134,8 @@ check_base(const char *tableList, const char *input, const char *expected,
 	int *outputPos = NULL;
 	int cursorPos = 0;
 	inbuf = malloc(sizeof(widechar) * inlen);
+	// + 1 : reserved for the '\0' terminator of C-format string argument used in
+	// `print_widechars`
```

Comment:
Before we merge we should squash this commit with c5668edea1920c966d6066626e968bb0bb217b35.

Reply:
Done